Gemfile

```diff
@@ -145,7 +145,7 @@ group :development do
 
   # Deployment
   gem 'capistrano3-delayed-job', '~> 1.0'
-  gem 'capistrano',  '~> 3.1'
+  gem 'capistrano',  '~> 3.10.1'
```

Comment:
This change shouldn't have been necessary to update to the latest version. Using `~>` with a two-digit version will always update to the latest minor version, in practice that means we would update to a `3.11` or `3.12` when they land, instead of being stuck on `3.10.x` until we manually change this version string again.

Reply:
I agree, I think `~> 3.10` is better